pkg/devfile/generator/utils.go

```diff
@@ -419,3 +419,46 @@ func getBuildConfigSpec(buildConfigSpecParams BuildConfigSpecParams) *buildv1.Bu
 		},
 	}
 }
+
+// GetVolumeMountPath gets the volume mount's path.
+func GetVolumeMountPath(volumeMount v1.VolumeMount) string {
```

Comment:
Any reason we want to make `GetVolumeMountPath` an exposed function?
non of the util functions are exposed. if it is expected to be exposed, then should be under generator.go

Reply:
odo is using it in some of their code, i will move it to generators.go